classes/battle/BattleManager.php

```diff
@@ -1185,7 +1184,7 @@ public function jutsuCollision(
         $player1_evasion_stat_amount = $this->getEvasionPercent($player1, $player1_jutsu, $player2->getBaseStatTotal());
         $player2_evasion_stat_amount = $this->getEvasionPercent($player2, $player2_jutsu, $player1->getBaseStatTotal());
 
-        if($player1_evasion_stat_amount >= $player2_evasion_stat_amount && $player2_jutsu_is_attack) {
+        if($player1_evasion_stat_amount >= $player2_evasion_stat_amount) {
```

Comment:
Actually curious about this one, why apply evasion against e.g. barriers? That would seem to make speed builds stronger than expected because they reduce the power of their opponent's barriers

Reply:
Normalizes barrier strength across different builds so that someone with 50%/50% BL/Off skill doesn't have barriers almost twice as strong as someone with 25%/25%/50% BL/Off/CSpd, even though normally both players do the same damage to each other with other jutsu.

Reply:
Ahh, I see. This seems pretty unintuitive from a "how does combat work" perspective, but I can see how it's simpler to balance this way